.github/workflows/pull-request.yml

```diff
@@ -34,29 +22,21 @@ jobs:
       - uses: actions/checkout@v3
       - uses: pnpm/action-setup@v2
         with:
-          version: latest
+          version: 'latest'
       - uses: actions/setup-node@v3
         with:
-          node-version: '16'
+          node-version: 'lts/gallium'
           cache: 'pnpm'
-      - run: pnpm install --frozen-lockfile
+      - run: pnpm install
       - run: pnpm build
 
-      - name: 🌳 Cypress run
-        uses: cypress-io/github-action@v4
-        with:
-          start: pnpm start:mocks
-          wait-on: 'http://localhost:8080'
+      - name: Cypress run
         env:
-          PORT: '8080'
-          SERVER_URL: http://localhost:8080
-          AUTH0_CLIENT_ID: ${{ secrets.AUTH0_CLIENT_ID }}
-          AUTH0_CLIENT_SECRET: ${{ secrets.AUTH0_CLIENT_SECRET }}
-          SESSION_SECRET: super-duper-s3cret
-          AUTH0_DOMAIN: nevados-dev.us.auth0.com
-          AUTH0_ORGANIZATION: org_3pmM22dWDm2Ppb95
-          AUTH0_TEST_USERNAME: test@nevados.solar
-          AUTH0_TEST_PASSWORD: test22!
+          GH_APP_ID: ${{ secrets.GH_APP_ID }}
```

Comment:
I am not sure if I am referencing the right secrets names because I don't have access to them. When they change, I expect the pipeline to pass